src/openforms/tests/e2e/test_input_validation.py

```diff
@@ -561,6 +561,7 @@ def test_required_field(self):
         self.assertValidationIsAligned(
             component,
             ui_input="",
+            api_value=None,
```

Comment:
I still don't find this intuitive and the inconsistent behaviour between tests is confusing. Additionally, the component type specific check in the generic base class is a smell and exactly where Formio.js went wrong in their SDK with their `Component` base class.

We can make this explicit quite easily - if we define a singleton in the base module and use that, we effectively decouple it while making it very readable:

in `input_validation_base.py`

```py
omit = object()

...

def _assertBackendValiation(self, ...):
    body_data = {} if value is omit else {key: value}
    body = {"data": body_data}
    ...
```

then here:

```py
from .input_validation_base import omit
```

and
```suggestion
            api_value=omit,
```

Reply:
Ah yeah thats a lot clearer